src/lib/net/address.h

```diff
@@ -209,6 +209,15 @@ tor_addr_eq_ipv4h(const tor_addr_t *a, uint32_t u)
  */
 #define TOR_ADDR_BUF_LEN 48
 
+/** Length of a buffer containing an IP address along with a port number and
+ * a seperating colon.
+ *
+ * This allows enough space for
+ *   "[ffff:ffff:ffff:ffff:ffff:ffff:255.255.255.255]:12345",
+ * plus a terminating NUL.
+ */
+#define TOR_ADDRPORT_BUF_LEN TOR_ADDR_BUF_LEN + 6
```

Comment:
Please include brackets in the macro definition. Otherwise, operators in the surrounding code could bind to parts of the macro definition, leading to an incorrect result.

For example, `TOR_ADDRPORT_BUF_LEN*2` becomes `TOR_ADDR_BUF_LEN + (6*2)`, which is not what we want.
```suggestion
#define TOR_ADDRPORT_BUF_LEN (TOR_ADDR_BUF_LEN + 6)
```